src/Nethermind/Ethereum.Test.Base/GeneralStateTest.cs

```diff
@@ -38,7 +38,7 @@ public class GeneralStateTest : IEthereumTest
         public UInt256? CurrentBaseFee { get; set; }
         public long CurrentGasLimit { get; set; }
         public long CurrentNumber { get; set; }
-        public UInt256 CurrentTimestamp { get; set; }
+        public ulong CurrentTimestamp { get; set; }
```

Comment:
Did we test backward compatibility? We changed type and I want to be sure that it will work for existing databses

Reply:
Yes we did. worked just fine.